packages/swapper/src/swappers/ChainflipSwapper/constants.ts

```diff
@@ -0,0 +1,53 @@
+import type { ChainId } from '@shapeshiftoss/caip'
+import { Asset, KnownChainIds } from "@shapeshiftoss/types";
+import { SupportedChainIds, SwapperName, SwapSource } from '../../types'
+
+export const ChainflipSupportedChainIds = [
+  KnownChainIds.EthereumMainnet,
+  KnownChainIds.ArbitrumMainnet,
+  KnownChainIds.BitcoinMainnet,
+  KnownChainIds.SolanaMainnet,
+  // TODO: Add Polkadot
+] as const
+
+export type ChainflipSupportedChainId = (typeof ChainflipSupportedChainIds)[number]
+
+export const ChainflipSupportedAssets = {
```

Comment:
```suggestion
export const ChainflipSupportedAssetIdsByChainId = {
```

As a general notion, we never speak symbol but always `AssetId` (`ChainflipSupportedAssets`) terminology, since those are chain-agnostic and contain the slip44, ChainId, and asset address/reference
. 
e.g for this specific const and throughout this PR, we should use AssetIds ([CAIP-19s](https://chainagnostic.org/CAIPs/caip-19)) here instead of symbols, you can find the `assetId` for each of these in `generatedAssetData.json` 🙏🏽

Reply:
Added a commit with temp CAIP-19s for the assets. I need a Zerion API key to generate that `generatedAssetData.json` file to get the correct ones

Reply:
Found them in the `caip` package!